### DISCUSSION TITLE
blockchain basic

### DISCUSSION THREAD

**Discussion by gibhub01** [Cyfrin/foundry-full-course-cu#2217] [Q&A]
on the BTC network do the block reward  and transaction fee go to the first miner only or distributed among the first 21 nodes who confirm the transaction with the first miner getting a bigger share

**Comment by urosognjenovic** [comment]
The mining reward, which is currently 3.125 BTC, and the transaction fess for that block go to the miner who successfully mined the block.

**Comment by EngrPips** [comment]
Interesting.

**Comment by lacsystem1411** [comment]
The number of confirmations depend on exchanges. For small transactions <$1,000 one confirmation is considered enough by most wallets; six confirmations in general for a transacction to be considered secured for amounts less than $1,000,000. 

Extracted from Btc white paper, I understand the payment goes to the first Node.

"By convention, the first transaction in a block is a special transaction that starts a new coin owned
by the creator of the block. This adds an incentive for nodes to support the network, and provides
a way to initially distribute coins into circulation, since there is no central authority to issue them.
The steady addition of a constant of amount of new coins is analogous to gold miners expending
resources to add gold to circulation. In our case, it is CPU time and electricity that is expended.
The incentive can also be funded with transaction fees. If the output value of a transaction is
less than its input value, the difference is a transaction fee that is added to the incentive value of
the block containing the transaction..."

**Reply by EngrPips** [reply]
Interesting. Thanks for this long explanation.
